R/helpers_ggbetween_t_subtitles.R

```diff
@@ -64,19 +64,20 @@ subtitle_t_parametric <- function(data,
       x = !!rlang::enquo(x),
       y = !!rlang::enquo(y)
     ) %>%
-    dplyr::mutate(.data = ., x = droplevels(as.factor(x))) %>%
+    dplyr::mutate_if(is.factor, droplevels) %>%
```

Comment:
Shouldn't the order of lines 67 and 68 opposite?

If `x` is of `character` type, then it needs to be converted to `factor` type and then the unused levels should be dropped?

Reply:
@ibecav This still needs to be resolved.

Reply:
I changed it but it really doesn't matter to my code anyway.  If we **_just_** turned a character into a factor then it's essentially impossible for there to be unused levels...

By the way lines 77-79 cost me an hour lol.  Took me awhile to figure out that `tidyr::gather` was turning right around and turning the factor we just constructed into a string again.  Since I'm relying on the fact that I'm being passed either two columns of numbers or one number one factor to sort out what to do. It was failing in a weird way.

      tidyr::gather(data = ., key, value, -rowid) %>%
      dplyr::rename(.data = ., x = key, y = value) %>%
      dplyr::mutate(x = factor(x))